manifests/dns_autoconf.pp

```diff
@@ -0,0 +1,67 @@
+#
```

Comment:
This *explicitly conflicts* with the usual `resolv` class.

You're going to need to do something to manage that conflict.